api/specs/webserver/openapi-projects.yaml

```diff
@@ -206,7 +206,7 @@ paths:
         required: false
         schema:
           type: boolean
-    post:
+    get:
```

Comment:
this should be a post. please check [google API guidelines](https://cloud.google.com/apis/design/custom_methods). we use that concept now everywhere else.

Reply:
We were not able to make the get_file_from_request work with the post method, that's why we changed to get. Blame @GitHK ;)

Reply:
I know what happened here 🙂. With the below implementation it is possible to use a POST on the download path.

The idea behind the GET was to let the browser automatically download the file without having to intercept nothing in javascript code. Since this was not what we have implemented I'll try and send a PR to have our desired standard.

Reply:
If you do so, try to go from /export to :export

Reply:
I really had no success with tat one 🙁, I will not try it again (already lost too much time on it). If someone can figure it out, please let me know.

Reply:
I think I have it working, where is my fullstack badge? :D

Reply:
Yes, everything is fixed now!